urdf/diffbot.urdf.xacro

```diff
@@ -1,30 +1,36 @@
 <?xml version="1.0"?>
-<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="mobile_robot">
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="diffbot">
 
-  <xacro:property name="package_name" value="mobile_robot_description"/>
+  <xacro:property name="package_name" value="diffbot_description"/>
```

Comment:
I think this parameter should be left as `mobile_robot_description` because is the name of the package and the path to find the files, isn't it?